templates/default/manage-disk-table.py.erb

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python
```

Comment:
Where is this file templated?

Reply:
This is a file for managing disk tables that Salman wrote. 
I added it now in the mgmd.rb script - will be created in the ndb/scripts folder for the mgmd host.